.stylelintrc

```diff
@@ -0,0 +1,39 @@
+{
+  "processors": ["stylelint-processor-styled-components"],
+  "extends": [
+    "stylelint-config-standard"
+  ],
+    "plugins": [
+    "stylelint-order"
+  ],
+  "rules": {
+    "font-family-name-quotes": ["always-unless-keyword"],
+    "function-url-quotes": "always",
+    "selector-attribute-quotes": "always",
+    "string-quotes": "single",
+    "selector-pseudo-class-no-unknown": [
+      true,
+      { "ignorePseudoClasses": ["global"] }
+    ],
+    "order/properties-order": [
+      ["composes"],
+      { "unspecified": "bottom" }
+    ],
+    "declaration-block-no-duplicate-properties": [
+      true,
+      { "ignoreProperties": ["composes"] }
+    ],
+    "property-no-unknown": [
+      true,
+      { "ignoreProperties": ["composes"] }
+    ],
+    "declaration-colon-newline-after": null,
+    "rule-empty-line-before": null,
+    "value-list-comma-newline-after": null,
+    "indentation": null,
+    "selector-descendant-combinator-no-non-space": null,
+    "no-descending-specificity": null,
+    "no-duplicate-selectors": null,
+    "no-empty-source": null
```

Comment:
I added this line because we are linting some files that ... don't have CSS. Like utils and such.

Reply:
The issue with excluding this is you would get an error from let's say, `parseTime.js` for having no css...